### ISSUE TITLE
Identify pages and properties missing in the WinForms experience (1+?)

### ISSUE THREAD

**Issue by tmeschter** [dotnet/project-system#6937]
We need to identify the WinForms-specific pages and properties missing from the new project property pages and file issues to address those.

**Comment by paul1956** [comment]
UseWindowsForms and UseWPF when you want to set both.
MinimumSplashScreenDisplayTime
Possibility UseCompatibleTextRendering

**Comment by tmeschter** [comment]
@paul1956 Are you requesting that we add controls for these? This work item is strictly about controls that exist on the current property pages but are missing from the new property pages.

**Comment by paul1956** [comment]
@tmeschter The package split of what is included between Framework and Core has changed. To get everything you got in Framework (by default) you need to set both options which was effectively done in Framework by doing nothing, WinForms should already be set for a WinForms application but WPF is not. Most applications don't need everything or even anything from the other so it seems reasonable to include a control, or to match Framework always set both whish is a really bad idea. Specifically Windows.Media is only in WPF but Windows.Media.Color is used on WinForm Applications and there is no way to set WPF.

MinimumSplashScreenDisplayTime was set manually in Framework and persisted. If Core if the project file is updated MinimumSplashScreenDisplayTime is deleted. It should be in the UI.

I don't know much about UseCompatibleTextRendering and how it was set but it was persisted in Framework if set to True.

**Comment by paul1956** [comment]
The first 2 can be set with the current UI but not the 3rd, all three come from Framework.
```
<AssemblyVersion>5.0.2.4</AssemblyVersion>
<FileVersion>5.0.2.4</FileVersion>
<Version>5.0.2.4</Version>
```

**Comment by paul1956** [comment]
Setting AssemblyVersion to * is flakey, sometime the original number is not deleted. This is very hard to consistently reproduce, it looks like a race.

**Comment by drewnoakes** [comment]
> UseWindowsForms and UseWPF when you want to set both.
> MinimumSplashScreenDisplayTime
> Possibility UseCompatibleTextRendering

- `UseWindowsForms` and `UseWPF` are tracked by #7239 and were implemented in #7281.
- `MinimumSplashScreenDisplayTime` and `UseCompatibleTextRendering` are historically only editable in a UI via the VB application model, which is tracked by #7236.
- There is a proposal in flight to unify these across languages in #7279.

**Comment by drewnoakes** [comment]
I have looked through the legacy UI in both a legacy C# project and an SDK-style C# project and cannot see any WinForms-specific properties to migrate to the new experience. The VB-specific items mentioned here are tracked by other issues.

Closing this.
